### PR TITLE
trainee start a subject within a course

### PR DESCRIPTION
**Pull request by duypn-2245**

### CODE REVIEW

**Comment by duypn-2245** [comment]
image
image
image
image

**Comment by duypn-2245** [comment]
https://edu-redmine.sun-asterisk.vn/issues/25752

**Comment by duypn-2245** [comment]
ready

File: app/models/user_subject.rb [hunk 1]
```diff
@@ -2,4 +2,6 @@ class UserSubject < ApplicationRecord
   enum status: {pending: 0, open: 1, finished: 2}
   belongs_to :user
   belongs_to :subject
+
+  scope :check_exist, ->(user_id){where "user_id = ?", user_id}
```
Comment (vanvtt-0952) [inline app/models/user_subject.rb]: `"user_id = ?", user_id ` -> `user_id: user_id`

**Comment by duypn-2245** [comment]
ready

**Pull request merged**